### PR TITLE
Fixed intranet lab plot. Included samples_map in GraphicJson

### LINKED ISSUE #157: Graph retains old data for empty laboratory selection
On the Intranet page, selecting a laboratory without samples does not update the graph. The previous data remains displayed instead of updating its state after the new selection .  In this scenario, it is required to make a graph reset or create an empty-state when no data is available.

### PR DESCRIPTION
**Pull request by Shettland**
This PR should speed up the time Intranet's sample map takes to load while also fixing #157 issue as now it will show that the lab has no data instead of not loading anything.

Closes #157

### CODE REVIEW

File: core/utils/samples.py [hunk 1]
```diff
@@ -758,7 +758,7 @@ def search_samples(sample_name, lab_name, sample_state, s_date, user):
         )
         sample_objs = sample_objs.filter(pk__in=state_ids)
     if s_date != "":
-        sample_objs = sample_objs.filter(sequencing_date_exact=s_date)
+        sample_objs = sample_objs.filter(sequencing_date=s_date)
```
Comment (saramonzon) [inline core/utils/samples.py]: Just curious, do you know why it's not working with exact?

File: dashboard/utils/generic_process_data.py [hunk 2]
```diff
@@ -455,3 +458,33 @@ def pre_proc_depth_sample_run():
...
+        data["ccaa_id"].append(region["properties"]["cartodb_id"])
+        data["ccaa_name"].append(ccaa_name)
+        if ccaa_name in raw_data["region"]:
+            data["samples"].append(raw_data["region"][ccaa_name])
+        else:
+            data["samples"].append("0")
+    dashboard.models.GraphicJsonFile.objects.create_new_graphic_json(
+        {"graphic_name": "received_samples_map", "graphic_data": data}
+    )
+    return {"SUCCESS": "Success"}
```
Comment (saramonzon) [inline dashboard/utils/generic_process_data.py:490]: we need to talk about this convention for the new refactor, I never liked this :(

File: core/utils/samples.py (hunk 1 above)
Comment (Shettland) [reply core/utils/samples.py]: Hmmmm could be that its `__iexact` instead of `_exact`

Comment (saramonzon) [reply core/utils/samples.py]: Yes! That's it! I knew something was off

**Review by saramonzon — APPROVED** [review]

**Pull request merged**